doc/version_history.rst

```diff
@@ -6,6 +6,13 @@
 Version History
 ===============
 
+v1.21.1
```

Comment:
1.21.0 was not released yet.. Maybe add these changes to it instead of creating v1.21.1?